core/lib/protobuf_config/src/api.rs

```diff
@@ -151,6 +152,8 @@ impl ProtoRepr for proto::Web3JsonRpc {
                 .context("whitelisted_tokens_for_aa")?,
             extended_api_tracing: self.extended_api_tracing.unwrap_or_default(),
             api_namespaces,
+            deployment_allowlist: read_required_repr(&self.deployment_allowlist)
+                .context("deployment_allowlist")?,
```

Comment:
The `deployment_allowlist` shouldn't be required; if it's missing, `DeploymentAllowlist::default()` should be used.